deny.toml

```diff
@@ -42,8 +42,10 @@ multiple-versions = "deny"
 wildcards = "deny"
 
 skip-tree = [
-    # rusoto_signature uses an older version of sha2
-    { name = "rusoto_signature" },
+    # h2 pulls in an older version of tokio util
+    { name = "h2",  version = "0.3.13" },
```

Comment:
This happens because the AWS SDK depends on both tokio and h2, and h2 depends on an older version of tokio?

Reply:
```
tokio-util v0.6.10
      └── aws-smithy-http v0.41.0
          ├── aws-config v0.11.0
          │   └── coldsnap v0.4.0
          ├── aws-endpoint v0.11.0
          │   ├── aws-sdk-ebs v0.11.0
          │   │   └── coldsnap v0.4.0 (*)
          │   ├── aws-sdk-ec2 v0.11.0
          │   │   └── coldsnap v0.4.0 (*)
          │   ├── aws-sdk-sso v0.11.0
          │   │   └── aws-config v0.11.0 (*)
          │   └── aws-sdk-sts v0.11.0
          │       └── aws-config v0.11.0 (*)
          ├── aws-http v0.11.0
          │   ├── aws-config v0.11.0 (*)
          │   ├── aws-sdk-ebs v0.11.0 (*)
          │   ├── aws-sdk-ec2 v0.11.0 (*)
          │   ├── aws-sdk-sso v0.11.0 (*)
          │   └── aws-sdk-sts v0.11.0 (*)
          ├── aws-sdk-ebs v0.11.0 (*)
          ├── aws-sdk-ec2 v0.11.0 (*)
          ├── aws-sdk-sso v0.11.0 (*)
          ├── aws-sdk-sts v0.11.0 (*)
          ├── aws-sig-auth v0.11.0
          │   ├── aws-sdk-ebs v0.11.0 (*)
          │   ├── aws-sdk-ec2 v0.11.0 (*)
          │   ├── aws-sdk-sso v0.11.0 (*)
          │   └── aws-sdk-sts v0.11.0 (*)
          ├── aws-sigv4 v0.11.0
          │   └── aws-sig-auth v0.11.0 (*)
          ├── aws-smithy-client v0.41.0
          │   ├── aws-config v0.11.0 (*)
          │   ├── aws-sdk-ebs v0.11.0 (*)
          │   ├── aws-sdk-ec2 v0.11.0 (*)
          │   ├── aws-sdk-sso v0.11.0 (*)
          │   ├── aws-sdk-sts v0.11.0 (*)
          │   └── aws-types v0.11.0
          │       ├── aws-config v0.11.0 (*)
          │       ├── aws-endpoint v0.11.0 (*)
          │       ├── aws-http v0.11.0 (*)
          │       ├── aws-sdk-ebs v0.11.0 (*)
          │       ├── aws-sdk-ec2 v0.11.0 (*)
          │       ├── aws-sdk-sso v0.11.0 (*)
          │       ├── aws-sdk-sts v0.11.0 (*)
          │       ├── aws-sig-auth v0.11.0 (*)
          │       └── coldsnap v0.4.0 (*)
          ├── aws-smithy-http-tower v0.41.0
          │   ├── aws-config v0.11.0 (*)
          │   ├── aws-sdk-ebs v0.11.0 (*)
          │   ├── aws-sdk-ec2 v0.11.0 (*)
          │   ├── aws-sdk-sso v0.11.0 (*)
          │   ├── aws-sdk-sts v0.11.0 (*)
          │   └── aws-smithy-client v0.41.0 (*)
          ├── aws-types v0.11.0 (*)
          └── coldsnap v0.4.0 (*)
    = tokio-util v0.7.2
      └── h2 v0.3.13
          └── hyper v0.14.18
              ├── aws-config v0.11.0
              │   └── coldsnap v0.4.0
              ├── aws-smithy-client v0.41.0
              │   ├── aws-config v0.11.0 (*)
              │   ├── aws-sdk-ebs v0.11.0
              │   │   └── coldsnap v0.4.0 (*)
              │   ├── aws-sdk-ec2 v0.11.0
              │   │   └── coldsnap v0.4.0 (*)
              │   ├── aws-sdk-sso v0.11.0
              │   │   └── aws-config v0.11.0 (*)
              │   ├── aws-sdk-sts v0.11.0
              │   │   └── aws-config v0.11.0 (*)
              │   └── aws-types v0.11.0
              │       ├── aws-config v0.11.0 (*)
              │       ├── aws-endpoint v0.11.0
              │       │   ├── aws-sdk-ebs v0.11.0 (*)
              │       │   ├── aws-sdk-ec2 v0.11.0 (*)
              │       │   ├── aws-sdk-sso v0.11.0 (*)
              │       │   └── aws-sdk-sts v0.11.0 (*)
              │       ├── aws-http v0.11.0
              │       │   ├── aws-config v0.11.0 (*)
              │       │   ├── aws-sdk-ebs v0.11.0 (*)
              │       │   ├── aws-sdk-ec2 v0.11.0 (*)
              │       │   ├── aws-sdk-sso v0.11.0 (*)
              │       │   └── aws-sdk-sts v0.11.0 (*)
              │       ├── aws-sdk-ebs v0.11.0 (*)
              │       ├── aws-sdk-ec2 v0.11.0 (*)
              │       ├── aws-sdk-sso v0.11.0 (*)
              │       ├── aws-sdk-sts v0.11.0 (*)
              │       ├── aws-sig-auth v0.11.0
              │       │   ├── aws-sdk-ebs v0.11.0 (*)
              │       │   ├── aws-sdk-ec2 v0.11.0 (*)
              │       │   ├── aws-sdk-sso v0.11.0 (*)
              │       │   └── aws-sdk-sts v0.11.0 (*)
              │       └── coldsnap v0.4.0 (*)
              ├── aws-smithy-http v0.41.0
              │   ├── aws-config v0.11.0 (*)
              │   ├── aws-endpoint v0.11.0 (*)
              │   ├── aws-http v0.11.0 (*)
              │   ├── aws-sdk-ebs v0.11.0 (*)
              │   ├── aws-sdk-ec2 v0.11.0 (*)
              │   ├── aws-sdk-sso v0.11.0 (*)
              │   ├── aws-sdk-sts v0.11.0 (*)
              │   ├── aws-sig-auth v0.11.0 (*)
              │   ├── aws-sigv4 v0.11.0
              │   │   └── aws-sig-auth v0.11.0 (*)
              │   ├── aws-smithy-client v0.41.0 (*)
              │   ├── aws-smithy-http-tower v0.41.0
              │   │   ├── aws-config v0.11.0 (*)
              │   │   ├── aws-sdk-ebs v0.11.0 (*)
              │   │   ├── aws-sdk-ec2 v0.11.0 (*)
              │   │   ├── aws-sdk-sso v0.11.0 (*)
              │   │   ├── aws-sdk-sts v0.11.0 (*)
              │   │   └── aws-smithy-client v0.41.0 (*)
              │   ├── aws-types v0.11.0 (*)
              │   └── coldsnap v0.4.0 (*)
              ├── hyper-rustls v0.22.1
              │   └── aws-smithy-client v0.41.0 (*)
              └── hyper-tls v0.5.0
                  └── aws-smithy-client v0.41.0 (*)
                  ```

Reply:
Looks like this was fixed in the recent 0.42.0 release: https://github.com/awslabs/aws-sdk-rust/commit/77e1c51891a654b49a9f49b0a25e55f9443bdecc.